### PR TITLE
I think I completed, as far as my understanding. Hit me up with bad code

### PR DESCRIPTION
**Pull request by Dancs777**
I don't know if it is good, just tell me what to fix and how.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 11
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233&bid=16085269)

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,43 @@ class Program
...
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Tell me if I did the task wrongly, because I wasn't really sure if I had had to define them and not ask them in.
+            //Read name, surename, age, weight (in kg) and height (in cm) from console
+            // calculate BMI index
+            string namePerson1 = "Tom";
+            string namePerson2 = "Daniel";
+            string surnamePerson1 = "Hartz";
+            string surnamePerson2 = "Lebrone";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'surnamePerson2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,43 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Tell me if I did the task wrongly, because I wasn't really sure if I had had to define them and not ask them in.
+            //Read name, surename, age, weight (in kg) and height (in cm) from console
+            // calculate BMI index
+            string namePerson1 = "Tom";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'namePerson1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            string namePerson2 = "Daniel";
+            string surnamePerson1 = "Hartz";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'surnamePerson1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            string surnamePerson2 = "Lebrone";
+            int agePerson1 = 32;
+            int agePerson2 = 53;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'agePerson2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -6,7 +6,43 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Tell me if I did the task wrongly, because I wasn't really sure if I had had to define them and not ask them in.
+            //Read name, surename, age, weight (in kg) and height (in cm) from console
+            // calculate BMI index
+            string namePerson1 = "Tom";
+            string namePerson2 = "Daniel";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'namePerson2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 6]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            {
+                Console.WriteLine("{0} {1} is {2} years old, his weight is {3} kg and his height {4} cm.", namePerson2, surnamePerson2, agePerson2, weightPerson2,heightPerson2);
+                Console.WriteLine("His BMI index is: {0}",bmiPerson2);
+                
+            }
+            else
+            {
+                Console.WriteLine("please select a valid number");
+                Console.WriteLine();
+                goto ifcycle;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this use of 'goto'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 7]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            string namePerson1 = "Tom";
+            string namePerson2 = "Daniel";
+            string surnamePerson1 = "Hartz";
+            string surnamePerson2 = "Lebrone";
+            int agePerson1 = 32;
+            int agePerson2 = 53;
+            int weightPerson1 = 82;
+            int weightPerson2 = 63;
+            double heightPerson1 = 181;
+            double heightPerson2 = 178;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'heightPerson2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 8]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            // calculate BMI index
+            string namePerson1 = "Tom";
+            string namePerson2 = "Daniel";
+            string surnamePerson1 = "Hartz";
+            string surnamePerson2 = "Lebrone";
+            int agePerson1 = 32;
+            int agePerson2 = 53;
+            int weightPerson1 = 82;
+            int weightPerson2 = 63;
+            double heightPerson1 = 181;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'heightPerson1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 9]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            //Read name, surename, age, weight (in kg) and height (in cm) from console
+            // calculate BMI index
+            string namePerson1 = "Tom";
+            string namePerson2 = "Daniel";
+            string surnamePerson1 = "Hartz";
+            string surnamePerson2 = "Lebrone";
+            int agePerson1 = 32;
+            int agePerson2 = 53;
+            int weightPerson1 = 82;
+            int weightPerson2 = 63;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'weightPerson2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 10]
```diff
@@ -6,7 +6,43 @@ class Program
...
+            //Tell me if I did the task wrongly, because I wasn't really sure if I had had to define them and not ask them in.
+            //Read name, surename, age, weight (in kg) and height (in cm) from console
+            // calculate BMI index
+            string namePerson1 = "Tom";
+            string namePerson2 = "Daniel";
+            string surnamePerson1 = "Hartz";
+            string surnamePerson2 = "Lebrone";
+            int agePerson1 = 32;
+            int agePerson2 = 53;
+            int weightPerson1 = 82;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'weightPerson1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs [hunk 11]
```diff
@@ -6,7 +6,43 @@ class Program
...
         {
-            Console.WriteLine("Hello World!");
+            //Tell me if I did the task wrongly, because I wasn't really sure if I had had to define them and not ask them in.
+            //Read name, surename, age, weight (in kg) and height (in cm) from console
+            // calculate BMI index
+            string namePerson1 = "Tom";
+            string namePerson2 = "Daniel";
+            string surnamePerson1 = "Hartz";
+            string surnamePerson2 = "Lebrone";
+            int agePerson1 = 32;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'agePerson1'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846233)

File: Src/BootCamp.Chapter/Program.cs (hunk 6 above)
Comment (Dancs777) [reply Src/BootCamp.Chapter/Program.cs]: What to do instead of it, to make sure you can only chose that one? use switch?

**Pull request closed without merging**